samples/csharp_dotnetcore/21.corebot-app-insights/CoreBot-App-Insights.csproj

```diff
@@ -14,4 +23,12 @@
     <PackageReference Include="Microsoft.Recognizers.Text.DataTypes.TimexExpression" Version="1.1.5" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I don't think we need to copy the luis model to output, consider removing this itemgroup (check original corebot for details

Reply:
Agreed. Removed.